buildconfig/pygame-stubs/camera.pyi

```diff
@@ -0,0 +1,17 @@
+from typing import List, Optional, Union, Tuple
+from pygame.surface import Surface
+
+
+def init() -> None: ...
+def quit() -> None: ...
+def list_cameras() -> List[str]: ...
+class Camera:
+    def __init__(self, device: Optional[int]=0, size: Optional[Union[Tuple[int, int], List[int]]]=(320, 200), format: Optional[str]="RGB"): ...
+    def start(self) -> None: ...
+    def stop(self) -> None: ...
+    def get_controls(self) -> Tuple[bool, bool, int]: ...
+    def set_controls(self, hflip: bool, vflip: bool, brightness: int) -> Tuple[bool, bool, int]: ...
```

Comment:
I think each of these arguments are optional. So this is possible: `set_controls(hflip=True)`. I'm not sure if this type captures that?

Reply:
Yes you are right, I missed that part from docs